### ISSUE TITLE
preload: TypeError: process.stdin.setRawMode is not a function

### ISSUE THREAD

**Issue by BrianAdams** [resin-io/resin-cli#637]
As of resin-cli 2ad789457f0a4c187912147519f0c9f04fa73522, the resin preload command is failing with `TypeError: process.stdin.setRawMode is not a function` just after it completes building the resin-preload container, while apparently trying to get the container size.

>resin preload $(pwd)/resin.img --dockerHost docker --dockerPort 2375 --app $appid --commit ${lastcommit}
```
...
Successfully tagged resin/resin-preload:latest
Fetching container size for rovimgupdatepreloadscript/b7be66e736b274e9344e7bd31c517e7e87fc3f1f
TypeError: process.stdin.setRawMode is not a function
    at runContainer (/usr/lib/node_modules/resin-cli/node_modules/resin-preload/lib/preload.js:109:17)
    at checkImage.then.then.then.then (/usr/lib/node_modules/resin-cli/node_modules/resin-preload/lib/preload.js:341:12)
    at tryCatcher (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/usr/lib/node_modules/resin-cli/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

**Comment by zvin** [comment]
@BrianAdams are you using windows?

**Comment by BrianAdams** [comment]
@zvin it is running on a linux ci server, inside a docker dind container.

**Comment by zvin** [comment]
@BrianAdams this issue is now fixed in resin-preload 2.0.3.
